.github/workflows/release-sdk-client.yml

```diff
@@ -0,0 +1,164 @@
+# This workflow exists because the client sdk build process requires macos.
+name: Manual Publish SDK Client Package
+on:
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        description: 'Is this a dry run. If so no package will be published.'
+        type: boolean
+        required: true
+  workflow_call:
+    inputs:
+      dry_run:
+        description: 'Is this a dry run. If so no package will be published.'
+        type: boolean
+        required: true
+    outputs:
+      hashes:
+        description: sha256sum hashes of built artifacts
+        value: ${{ jobs.publish.outputs.hashes }}
+
+jobs:
+# Building is done on mac runner due to xcode build dependencies
+  build:
+    runs-on: macos-latest
+    permissions:
+      id-token: write
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # If you only need the current version keep this
+
+      - name: Setup Env from project's Env file
+        shell: bash
+        run: echo "$(cat pkgs/sdk/client/github_actions.env)" >> $GITHUB_ENV
+
+      - uses: launchdarkly/gh-actions/actions/release-secrets@release-secrets-v1.0.1
+        name: Get secrets
+        with:
+          aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+          s3_path_pairs: 'launchdarkly-releaser/dotnet/LaunchDarkly.snk = LaunchDarkly.snk'
+
+      - name: CI check
+        uses: ./.github/actions/ci
+        with:
+          project_file: ${{ env.PROJECT_FILE }}
+          test_project_file: ${{ env.TEST_PROJECT_FILE }}
+
+      - name: Release build
+        uses: ./.github/actions/build-release
+        with:
+          project_file: ${{ env.PROJECT_FILE }}
+
+      - name: Build Documentation
+        uses: ./.github/actions/build-docs
+        with:
+          workspace_path: ${{ env.WORKSPACE_PATH }}
+
+      - name: Retain build artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: dir-bin-release
+          path: ${{ env.BUILD_OUTPUT_PATH }}
+          retention-days: 1
+
+      - name: Retain docs artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: dir-docs
+          path: ${{ env.WORKSPACE_PATH }}/docs
+          retention-days: 1
+
+# Signing DLLs is done on Ubuntu due to Digicert tooling compatibility
+  sign-dlls:
+    runs-on: ubuntu-latest
+    needs: build
+    permissions:
+      id-token: write
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
```

Comment:
This one could probably drop the `with` section.

Reply:
Whichever build is the one doing docs is the one that may need the 0.

Reply:
Done